pkg/cloudprovider/OWNERS

```diff
@@ -3,7 +3,6 @@
 approvers:
   - mikedanese
   - dims
-  - wlan0
```

Comment:
@liggitt i use this url (with Repository group "Kubernetes" instead of "All)
https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=Kubernetes&var-repo_name=kubernetes%2Fkubernetes&var-country_name=All

Reply:
i'll add wlan0 back for now